sdk/storage/mgmt/src/main/java/com/azure/management/storage/StorageManagementClientBuilder.java

```diff
@@ -96,11 +85,11 @@ public StorageManagementClientBuilder pipeline(HttpPipeline pipeline) {
     }
 
     /**
-     * Builds an instance of StorageManagementClientImpl with the provided parameters.
+     * Builds an instance of StorageManagementClient with the provided parameters.
      *
-     * @return an instance of StorageManagementClientImpl.
+     * @return an instance of StorageManagementClient.
      */
-    public StorageManagementClientImpl buildClient() {
+    public StorageManagementClient buildClient() {
         if (host == null) {
             this.host = "https://management.azure.com";
```

Comment:
In track 2, we've named this as endpoint instead of `host`.

Reply:
I guess it is in swagger as well https://github.com/Azure/azure-rest-api-specs/blob/master/specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/blob.json#L8

I will see if I can rename it in generator for mgmt.